app/views/payments/index.html.erb

```diff
@@ -0,0 +1,17 @@
+<div class='form-head'>
+    <%= link_to categories_path do %>
+      <i class="fa fa-arrow-left"></i>
+    <% end %>
+    <h2>Transactions</h2>
+    <div class="actions">
+      <%= button_to "Sign Out", destroy_user_session_path, method: :delete, class:"btn",style: 'color:white; background-color:transparent;' %>
+    </div>
+  </div>
+<div>
+<div id="list-con">
+  <% @payments.each do |payment| %>
+    <%= render payment %>
+  <% end %>
+</div>
+
+<%= link_to "New payment", new_category_payment_path, class: 'btn w-100 p-2', style: 'color:white; background-color:#5fb523;'%>
```

Comment:
- [ ] Please move the "New payment" button to the bottom of the screen to match the given design.
- [ ] Please add a white outline around the button to match the original button design.

| Your design | Original design |
| --- | --- |
| ![image](https://github.com/Kccendana/budget_App/assets/62290545/99badc8a-2448-46f8-92f5-0a0f9c314244) | ![image](https://github.com/Kccendana/budget_App/assets/62290545/c36f0799-7be4-4890-b7c1-dddc36077372) |